modules/ossm-routing-ingress.adoc

```diff
@@ -25,7 +25,7 @@ If the `EXTERNAL-IP` value is set, your environment has an external load balance
 
 If the `EXTERNAL-IP` value is `<none>`, or perpetually `<pending>`, your environment does not provide an external load balancer for the ingress gateway.
 ifdef::openshift-enterprise[]
-You can access the gateway using the service's xref:../../networking/configuring-node-port-service-range.adoc[node port].
+You can access the gateway using the service's link:https://docs.openshift.com/container-platform/latest/networking/configuring-node-port-service-range.html#configuring-node-port-service-range[node port].
```

Comment:
We need to swap from links that link to our internal docs to xrefs. The reason for this is that, if customers are on the portal and they click this docs.openshift.com link, they'll be redirected to docs.opc.com. this is a bad UX. These xrefs will need to be put into the associated assembly, since xrefs cannot go into modules.

Reply:
@smunje1 - It's incorrect here to use either an xref or a hard link. Both are incorrect. We can't use xrefs in modules, and we can't hard link to docs.openshift or access.redhat to circumvent this restriction. So, this needs to be changed back to an xref with a correct path even though using an xref is also incorrect.